packages/graphql/src/services/__tests__/graphql.service.spec.ts

```diff
@@ -416,6 +416,15 @@ describe('GraphqlService', () => {
 
       expect(removeSpaces(query)).toBe(removeSpaces(expectation));
     });
+
+    it('should return a query string including a direct reference to a complex object', () => {
+      const expectation = `firstName, lastName, billing{address{street, zip}}`;
+      const columns = ['firstName', 'lastName', 'billing', 'billing.address.street', 'billing.address.zip'];
```

Comment:
so the only difference is that you added `billing` and this was previously throwing an error, is that it?

Reply:
Yeah, that's right